### PR TITLE
Add overlays for Pytorch CRDS; update Pytorch version

### LINKED ISSUE #483: add application overlay for pytorch-job-crds

### PR DESCRIPTION
**Pull request by richardsliu**
**Which issue is resolved by this Pull Request:**
Resolves #483 

**Description of your changes:**

**Checklist:**
- [ ] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate`
    3. `make test`

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/504)

### CODE REVIEW

File: pytorch-job/pytorch-job-crds/overlays/application/application.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      app.kubernetes.io/component: pytorch
+      app.kubernetes.io/part-of: kubeflow
+      app.kubernetes.io/managed-by: kfctl
+  componentKinds:
+  - group: core
+    kind: Service
+  - group: apps
+    kind: Deployment
+  - group: core
+    kind: ConfigMap
```
Comment (johnugeorge) [inline pytorch-job/pytorch-job-crds/overlays/application/application.yaml]: Manifests don't have a ConfigMap?

Comment (kkasravi) [reply pytorch-job/pytorch-job-crds/overlays/application/application.yaml]: @johnugeorge you're right. There are cases where we do a configMapGenerator, but not here.
@richardsliu could you remove lines 19,20?
if you use the gen-tests-targets.sh from #466 (eg copy to your ~)

~/gen-test-targets pytorch-job/pytorch-job-crds/overlays/application

**Comment by richardsliu** [comment]
@kkasravi Thanks, this is much faster.

**Pull request merged**